libsolidity/formal/CHC.cpp

```diff
@@ -61,11 +61,62 @@ bool CHC::visit(ContractDefinition const& _contract)
 	if (!shouldVisit(_contract))
 		return false;
 
+	m_contract = &_contract;
+
 	reset();
 
 	if (!SMTEncoder::visit(_contract))
 		return false;
 
+	for (auto const& contract: _contract.annotation().linearizedBaseContracts)
+		for (auto var: contract->stateVariables())
+			if (*contract == _contract || var->isVisibleInDerivedContracts())
+				m_stateVariables.push_back(var);
+
+	for (auto const& var: m_stateVariables)
```

Comment:
Same here: `updateStateSorts` ? Or is this function going to be refactored anyways?

Reply:
Here I'm not sure, since it's not used anywhere else

Reply:
ok!